pkgs/top-level/all-packages.nix

```diff
@@ -6485,7 +6485,9 @@ with pkgs;
 
   keyfuzz = callPackage ../tools/inputmethods/keyfuzz { };
 
-  keystore-explorer = callPackage ../applications/misc/keystore-explorer { };
+  keystore-explorer = callPackage ../applications/misc/keystore-explorer {
+    jdk = jdk11;
```

Comment:
I'm personally not a fan of this pattern, since `jdk` already has a default value (other than `jdk11`). It can be a bit confusing when reading the `default.nix` and assuming that the package uses the default `jdk` version.

[Apparently](https://discourse.nixos.org/t/package-version-arguments-vs-generic-package-arguments/9071/7) there is no clear consensus though, so your opinion is as good as mine. Just wanted to mention it :)